### ISSUE TITLE
Can't feed result of get operation into feed delete/update

### ISSUE THREAD

**Issue by PrathamSoni** [vespa-engine/pyvespa#758]
feed_iterable requires ids to be formatted as just last component without namespace etc, but get operation returns the raw document id from vespa.

**Comment by thomasht86** [comment]
Good point. You could of course infer the id from the doc_id for now, but we'll see if we can make this easier in the future.
